runtime/common/src/xcm_impl.rs

```diff
@@ -35,8 +37,21 @@ use xcm_executor::{
 	Assets,
 };
 
-pub fn native_currency_location(para_id: u32, id: CurrencyId) -> MultiLocation {
-	MultiLocation::new(1, X2(Parachain(para_id), GeneralKey(id.encode())))
+pub fn local_currency_location(key: Vec<u8>) -> MultiLocation {
```

Comment:
local currencies can always do `encode` inside the function right?

Reply:
yeah, I'm considering align to `native_currency_location` which both use `Vec<u8>` as parameter, but of course `local_currency_location` can use `CurrencyId` as parameter, and doing `encode` inside.

Reply:
it's not necessary to keep them consistent, they are not same types of currencies anyway